src/ui/Facet/FacetSettings.ts

```diff
@@ -56,6 +56,10 @@ export class FacetSettings extends FacetSort {
     this.settingsButton.setAttribute('aria-expanded', `${expanded}`);
   }
 
+  private get firstFocusablePopupElement() {
+    return find(Dom.nodeListToArray(this.settingsPopup.querySelectorAll('[tabindex]')), element => element.tabIndex >= 0);
```

Comment:
Use `findAll` from Dom utility.

```
find($$(this.settingsPopups).findAll('[tabindex]'), .....
```

Reply:
That looks a lot better! I'll use that instead.